lib/init.g

```diff
@@ -311,6 +311,7 @@ BIND_GLOBAL("ReadAndCheckFunc",function( arg )
         local  name,  ext,  libname, error;
 
         name := arg[1];
+
```

Comment:
Why is this newline being inserted?

Reply:
I have a habit of visually separating comments that way because i dont like comments in the middle of code blocks.

Reply:
I don't mind it much, as long as it is equally applied to both copies of the code -- what's bad is if new diffs between them are introduced.